node/accounts-bot/index.js

```diff
@@ -0,0 +1,183 @@
+const {google} = require('googleapis');
+const SHEETS_SCOPE = 'https://www.googleapis.com/auth/spreadsheets.readonly';
+const ACCOUNTS_SHEET_ID = '1kxW15ZI48mh4KkvgsMpg7gInmEQmyKYRnZdbUOSMRnU';
+const ACCOUNTS_SHEET_URL = 'https://docs.google.com/spreadsheets/d/1kxW15ZI48mh4KkvgsMpg7gInmEQmyKYRnZdbUOSMRnU/edit#gid=0';
+const ACCOUNT_IMAGE_URL = 'https://www.gstatic.com/images/icons/material/system_gm/1x/account_circle_black_18dp.png';
+
+/**
+ * Looks up the account owner for the company requested.
+ *
+ * @param {!express:Request} req HTTP request context.
+ * @param {!express:Response} res HTTP response context.
+*/
+exports.accountsBot = async (req, res) => {
+  const message = req.query.message || req.body.message;
```

Comment:
Really should verify the auth header. We haven't had this in our samples, but should. Otherwise the bot would expose the sales data if somebody was able to figure out the underlying URL.

Reply:
Filed #73 to do it after writing the Node guide for it